phpScripts/editAboutMe.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+
```

Comment:
Now this is SUPER picky, but just in case charlie mentions, theres a few extra lines in between these functions!